### PR TITLE
Add memory limit section to PHP

### PR DESCRIPTION
**Pull request by gaelreyrol**
I added a new section to describe how php memory limit is defined and how a user can change it.

### CODE REVIEW

File: deploy/application/php/php-apps.md [hunk 1]
```diff
@@ -16,6 +16,28 @@ PHP is available on our platform with the branches 5.6, 7.2, 7.3, 7.4 and 8.0. Y
 
 The HTTP server is [Apache 2](https://httpd.apache.org/), and the PHP code is executed by [PHP-FPM](https://php-fpm.org/).
 
+## Memory Limit
```
Comment (BlackYoup) [inline deploy/application/php/php-apps.md:19]: This section should be put later in the documentation, maybe after "Configure your PHP application"? Maybe it doesn't even require a `h2` title, just another sub-section.

File: deploy/application/php/php-apps.md [hunk 2]
```diff
@@ -16,6 +16,28 @@ PHP is available on our platform with the branches 5.6, 7.2, 7.3, 7.4 and 8.0. Y
...
+
+A smaller part of your application's memory is defined as the memory limit to allow workers to run efficiently, here is the memory limit for each flavor:
```
Comment (BlackYoup) [inline deploy/application/php/php-apps.md:21]: I'm not sure to really understand what's this limit is when I read this sentence. I feel like this says `we only use 2048M of memory even though you have 32GB in total`. But that may be only me.

IIRC, this is the memory limit of each worker spawned by the `php-fpm` process.

**Review by BlackYoup — CHANGES_REQUESTED** [review]

**Comment by gaelreyrol** [comment]
It should be better ?

**Comment by BlackYoup** [comment]
Thanks :)

**Pull request merged**